src/Microsoft.ComponentDetection.Detectors/go/GoComponentWithReplaceDetector.cs

```diff
@@ -33,7 +33,7 @@ public class GoComponentWithReplaceDetector : FileComponentDetector, IExperiment
         IObservableDirectoryWalkerFactory walkerFactory,
         ICommandLineInvocationService commandLineInvocationService,
         IEnvironmentVariableService envVarService,
-        ILogger<GoComponentDetector> logger)
+        ILogger<GoComponentWithReplaceDetector> logger)
     {
```

Comment:
are you keeping the detector version from previous PR or do we need to bump it again?

Reply:
same version